Extension/src/LanguageServer/client.ts

```diff
@@ -1595,10 +1595,10 @@ export class DefaultClient implements Client {
         if (document.uri.scheme === "file") {
             const uri: string = document.uri.toString();
             openFileVersions.set(uri, document.version);
-            vscode.commands.executeCommand('setContext', 'BuildAndDebug.isSourceFile', util.isCppOrCFile(document.uri));
-            vscode.commands.executeCommand('setContext', 'BuildAndDebug.isFolderOpen', util.isFolderOpen(document.uri));
+            vscode.commands.executeCommand('setContext', 'cpptools.BuildAndDebug.isSourceFile', util.isCppOrCFile(document.uri));
```

Comment:
Do you want to take this opportunity to apply naming conventions on `BuildAndDebug` while you're at it?